features/dd-sdk-android-trace/src/main/java/java8/util/function/BiConsumer.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+/*
+ * Copyright (c) 2012, 2013, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
```

Comment:
important: I think we may hit an issue here since it is GPLv2 license. So if we copy and modify source code from Oracle JDK that is under GPLv2, we must release that modified code under GPLv2. But to be checked.

Reply:
Oh...I copy pasted these files so I assumed that it will not add our licence automatically, need to remove that part. Also I wanted to double check in the daily if for these files we are good if we copy paste them together with the License header also. @xgouchet any hints ?

Reply:
if it is fine to **copy** these files, this copyright should be kept

Reply:
yes but I don't think we should keep ours that was automatically added in the header of the file. I think we should only keep theirs as this code does not belong us. I removed our header but will still need to discuss it further to make sure we are 100% safe.